### PR TITLE
Added missing schemas for ReconstructedCell to atlas space transformation

### PR DESCRIPTION
**Pull request by annakristinkaufmann**

### CODE REVIEW

File: modules/morphology/src/main/resources/schemas/neurosciencegraph/morphology/fixationstainingmounting/v0.1.1.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+              "path": "nsg:mountingMedia",
+              "name": "Mounting Media",
+              "description": "Mounting media is the medium that the sample is in while it is being imaged on the microscope such as Xylene or Mowiol.",
+              "datatype": "xsd:string",
+              "maxCount": 1
+            },
+            {
+              "path": "prov:used",
+              "description": "The patched slice used",
+              "xone": [
```
Comment (MFSY) [inline modules/morphology/src/main/resources/schemas/neurosciencegraph/morphology/fixationstainingmounting/v0.1.1.json]: Looks like a cardinality difference. I don't think this should be handled by xone.

File: modules/morphology/src/main/resources/schemas/neurosciencegraph/morphology/fixationstainingmounting/v0.1.1.json [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                }
+              ],
+              "minCount": 1,
+              "maxCount": 1,
+              "seeAlso": "{{base}}/schemas/neurosciencegraph/experiment/patchedslice/v0.1.0/shapes/PatchedSliceShape"
+            },
+            {
+              "path": "prov:generated",
+              "description": "The fixed and stained slice used",
+              "xone": [
```
Comment (MFSY) [inline modules/morphology/src/main/resources/schemas/neurosciencegraph/morphology/fixationstainingmounting/v0.1.1.json]: Looks like a cardinality difference. I don't think this should be handled by xone.

File: modules/atlas/src/test/resources/data/neurosciencegraph/atlas/transform/v0.1.0/min-fields.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/data/v0.1.0"
+  ],
+  "@type": "nsg:Transform",
+  "@id": "{{base}}/data/neurosciencegraph/atlas/transform/v0.1.0/4315069b-9e89-4473-a965-940c7d54d7c5",
+  "name": "",
+  "distribution": {
+    "accessURL": "file:///projectpath/210254/transformation/affine.h5"
```
Comment (MFSY) [inline modules/atlas/src/test/resources/data/neurosciencegraph/atlas/transform/v0.1.0/min-fields.json]: Use a downloadURL instead and use http.

File: modules/experiment/src/main/resources/schemas/neurosciencegraph/experiment/brainimaging/v0.1.0.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/morphology/brainimaging/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0",
```
Comment (MFSY) [inline modules/experiment/src/main/resources/schemas/neurosciencegraph/experiment/brainimaging/v0.1.0.json]: Update to neurosciencegraph/commons/activity/v0.1.3

File: modules/atlas/src/main/resources/schemas/neurosciencegraph/atlas/transformation/v0.1.0.json [hunk 5]
```diff
@@ -0,0 +1,62 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/atlas/transformation/v0.1.0/shapes/"
+    }
+  ],
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0"
```
Comment (MFSY) [inline modules/atlas/src/main/resources/schemas/neurosciencegraph/atlas/transformation/v0.1.0.json]: Use the [neurosciencegraph/commons/activity/v0.1.3](https://github.com/INCF/neuroshapes/blob/master/modules/commons/src/main/resources/schemas/neurosciencegraph/commons/activity/v0.1.3.json) instead

File: modules/experiment/src/main/resources/schemas/neurosciencegraph/experiment/brainslicing/v0.1.1.json [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+            },
+            {
+              "path": "nsg:cuttingThickness",
+              "name": "Cutting thickness",
+              "description": "Cutting thickness of brain slice",
+              "node": "{{base}}/schemas/neurosciencegraph/commons/quantitativevalue/v0.1.0/shapes/QuantitativeValueShape",
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:hemisphere",
```
Comment (MFSY) [inline modules/experiment/src/main/resources/schemas/neurosciencegraph/experiment/brainslicing/v0.1.1.json]: I would suggest to use brainHemisphere here.

File: modules/experiment/src/main/resources/schemas/neurosciencegraph/experiment/brainslicing/v0.1.1.json [hunk 7]
```diff
@@ -0,0 +1,101 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/experiment/brainslicing/v0.1.0/shapes/"
+    }
+  ],
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.2",
```
Comment (MFSY) [inline modules/experiment/src/main/resources/schemas/neurosciencegraph/experiment/brainslicing/v0.1.1.json]: Use the [neurosciencegraph/commons/activity/v0.1.3](https://github.com/INCF/neuroshapes/blob/master/modules/commons/src/main/resources/schemas/neurosciencegraph/commons/activity/v0.1.3.json) schema instead

File: modules/experiment/src/main/resources/schemas/neurosciencegraph/experiment/brainslicing/v0.1.1.json [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+    "{{base}}/schemas/neurosciencegraph/commons/quantitativevalue/v0.1.0",
```
Comment (MFSY) [inline modules/experiment/src/main/resources/schemas/neurosciencegraph/experiment/brainslicing/v0.1.1.json]: Use the 
[neurosciencegraph/commons/quantitativevalue/v0.1.2](https://github.com/INCF/neuroshapes/blob/master/modules/commons/src/main/resources/schemas/neurosciencegraph/commons/quantitativevalue/v0.1.2.json) schema instead. Or you can remove this import as the neurosciencegraph/commons/quantitativevalue/v0.1.2 is already imported through the [neurosciencegraph/commons/activity/v0.1.3](https://github.com/INCF/neuroshapes/blob/master/modules/commons/src/main/resources/schemas/neurosciencegraph/commons/activity/v0.1.3.json).

File: modules/experiment/src/main/resources/schemas/neurosciencegraph/experiment/brainslicing/v0.1.1.json [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+    "{{base}}/schemas/neurosciencegraph/commons/brainlocation/v0.1.0"
```
Comment (MFSY) [inline modules/experiment/src/main/resources/schemas/neurosciencegraph/experiment/brainslicing/v0.1.1.json]: Use the 
[neurosciencegraph/commons/brainlocation/v0.1.1](https://github.com/INCF/neuroshapes/blob/master/modules/commons/src/main/resources/schemas/neurosciencegraph/commons/brainlocation/v0.1.1.json) schema instead.

File: modules/experiment/src/main/resources/schemas/neurosciencegraph/experiment/brainslicing/v0.1.1.json [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+              "in": [
+                "Left",
+                "Right"
+              ],
+              "maxCount": 2
+            },
+            {
+              "path": "nsg:solution",
+              "name": "Solution",
+              "description": "Solution in which the brain was sliced",
```
[the remark this answers is not in the document]
Comment (annakristinkaufmann) [reply modules/experiment/src/main/resources/schemas/neurosciencegraph/experiment/brainslicing/v0.1.1.json]: An example would be Ringer's solution (https://en.wikipedia.org/wiki/Ringer%27s_solution)

File: modules/experiment/src/main/resources/schemas/neurosciencegraph/experiment/slicecollection/v0.1.0.json [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/emptycollection/v0.1.0/shapes/EmptyCollectionShape"
+        },
+        {
+          "property": [
+            {
+              "path": "prov:hasPart",
+              "name": "Has part",
+              "description": "Labeled cell",
+              "class": "nsg:LabeledCell",
```
[the remark this answers is not in the document]
Comment (annakristinkaufmann) [reply modules/experiment/src/main/resources/schemas/neurosciencegraph/experiment/slicecollection/v0.1.0.json]: labeled cells which are part of the slice collection (one labeled cell spans multiple slices)

File: modules/experiment/src/main/resources/schemas/neurosciencegraph/experiment/brainslicing/v0.1.1.json (hunk 10 above)
[the remark this answers is not in the document]
Comment (MFSY) [reply modules/experiment/src/main/resources/schemas/neurosciencegraph/experiment/brainslicing/v0.1.1.json]: Yep. I was thinking of adding it in the description.

File: modules/atlas/src/main/resources/schemas/neurosciencegraph/atlas/transform/v0.1.0.json [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "targetClass": "nsg:Transform",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0/shapes/EntityShape"
+        },
+        {
+          "property": [
+            {
+              "path": "schema:distribution",
```
[the remark this answers is not in the document]
Comment (annakristinkaufmann) [reply modules/atlas/src/main/resources/schemas/neurosciencegraph/atlas/transform/v0.1.0.json]: Done

File: modules/atlas/src/main/resources/schemas/neurosciencegraph/atlas/transformation/v0.1.0.json (hunk 5 above)
Comment (annakristinkaufmann) [reply modules/atlas/src/main/resources/schemas/neurosciencegraph/atlas/transformation/v0.1.0.json]: Done

File: modules/experiment/src/main/resources/schemas/neurosciencegraph/experiment/brainimaging/v0.1.0.json (hunk 4 above)
Comment (annakristinkaufmann) [reply modules/experiment/src/main/resources/schemas/neurosciencegraph/experiment/brainimaging/v0.1.0.json]: Done

File: modules/morphology/src/main/resources/schemas/neurosciencegraph/morphology/reconstructioncorrection/v0.1.0.json [hunk 13]
```diff
@@ -0,0 +1,43 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/morphology/reconstructioncorrection/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0",
```
Comment (MFSY) [inline modules/morphology/src/main/resources/schemas/neurosciencegraph/morphology/reconstructioncorrection/v0.1.0.json]: I suggest to use neurosciencegraph/commons/activity/v0.1.4

File: modules/morphology/src/main/resources/schemas/neurosciencegraph/morphology/reconstructioncorrection/v0.1.0.json [hunk 14]
```diff
@@ -0,0 +1,43 @@
...
+    "{{base}}/schemas/neurosciencegraph/commons/typedlabeledontologyterm/v0.1.0"
```
Comment (MFSY) [inline modules/morphology/src/main/resources/schemas/neurosciencegraph/morphology/reconstructioncorrection/v0.1.0.json]: I suggest to use neurosciencegraph/commons/typedlabeledontologyterm/v0.1.2

File: modules/morphology/src/main/resources/schemas/neurosciencegraph/morphology/reconstructioncorrection/v0.1.0.json [hunk 15]
```diff
@@ -0,0 +1,43 @@
...
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0/shapes/ActivityShape"
+        },
+        {
+          "property": [
+            {
+              "path": "prov:used",
+              "class": "nsg:ReconstructedWholeBrainCell",
```
Comment (MFSY) [inline modules/morphology/src/main/resources/schemas/neurosciencegraph/morphology/reconstructioncorrection/v0.1.0.json]: This class doesn't match what is in the prov template below. A ReconstructionCorrection takes an instance of nsg:ReconstructedCell and generates a revision of it which is an instance nsg:ReconstructedCell. 
Can this schema be aligned with the prov template ?

File: provpatterns/assets/transform-morphology-atlas-space-prov-template.svg [hunk 16]
```diff
@@ -1,2 +1,2 @@
 <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```
Comment (MFSY) [inline provpatterns/assets/transform-morphology-atlas-space-prov-template.svg]: I suggest to use nsg:Reconstructionfromimage as a reconstruction activity and not nsg:Reconstrcution

File: modules/atlas/src/main/resources/schemas/neurosciencegraph/atlas/transformation/v0.1.0.json [hunk 17]
```diff
@@ -0,0 +1,63 @@
...
+  ],
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.4"
+  ],
+  "@type": "nxv:Schema",
+  "shapes": [
+    {
+      "@id": "this:TransformationShape",
+      "@type": "sh:NodeShape",
+      "description": "Transformation activity shape description.",
```
[the remark this answers is not in the document]
Comment (annakristinkaufmann) [reply modules/atlas/src/main/resources/schemas/neurosciencegraph/atlas/transformation/v0.1.0.json]: Done

File: provpatterns/assets/transform-morphology-atlas-space-prov-template.svg (hunk 16 above)
Comment (annakristinkaufmann) [reply provpatterns/assets/transform-morphology-atlas-space-prov-template.svg]: Done

**Pull request merged**